### PR TITLE
Removed deprecated API method and code warning, updated to Swift 5

### PR DESCRIPTION
**Pull request by domeniconicoli**
-Removed deprecated API methods
-Updated to Swift 5 version
-Removed many code warnings

### CODE REVIEW

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: could you remove this change?

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: this one too

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: ditto, delete please

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: all of the ones that add your development team, there are a couple more in the file

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: is this required?

**Review by AndrewSB — CHANGES_REQUESTED** [review]
this is super helpful, thanks for taking the time. we can bump the version after this PR is merged. take a look at my notes and I’m happy to merge after you’ve made the changes I requested

**Comment by domeniconicoli** [comment]
Hi Andrew,
I made the requested changes.
Let me know if everything is fine now.

Best Regards.

**Comment by AndrewSB** [comment]
pushed a new release: https://github.com/zhxnlai/ZLSwipeableViewSwift/releases/tag/0.0.9

**Pull request merged**
